tests/unit/test_graph.py

```diff
@@ -140,6 +141,7 @@ def get_config(self, extra_cfg=None):
 
         config = config_from_parts_or_dicts(project=cfg, profile=self.profile)
         dbt.flags.set_from_args(Namespace(), config)
+        setup_event_logger(dbt.flags.get_flags())
```

Comment:
This change is because test_graph was depending on some other test setup event logger